static/styles/drafts.css

```diff
@@ -73,8 +73,6 @@
 }
 
 .draft-row {
-    padding: 5px 25px;
```

Comment:
We should keep the vertical padding here, otherwise it moves individual drafts closer than we want them to be.

Reply:
Does the commit message from 714c29c658f89386464db977f87ac74d782863aa change your mind? I wanted to move the padding elsewhere because I was adding more items to the `drafts-list` container

Reply:
This does have a visual effect of reduced vertical padding between individual drafts. If we are fine with how it looks after, I don't think should be any issue.

Reply:
Oh!! I see what you mean now, yeah. I think I did remove the vertical padding on purpose to look like the designs from @terpimost, but forgot because too much time had passed 😅 

I should update the commit message to say that there **is** a visual difference. But also -- Vlad can you confirm the padding looks okay?

Thanks Aman :)

Reply:
Ok I'm honestly not sure if I'd meant to change the padding, so I'm going to do what you said and preserve the vertical pattern unless Vlad would prefer something else.